digitalforms-api/src/main/java/ca/bc/gov/open/pssg/rsbc/digitalforms/controller/ApplicationFormController.java

```diff
@@ -44,64 +47,100 @@
 	@Autowired
 	ApplicationFormService service;
 
-	@GetMapping(value = "/{formType}/{GUID}/application", produces = DigitalFormsConstants.JSON_CONTENT)
+	Logger logger = LoggerFactory.getLogger(ApplicationFormController.class);
+
+	@GetMapping(value = "/{formType}/{GUID}/application/{requestGuid}", produces = DigitalFormsConstants.JSON_CONTENT)
```

Comment:
If I'm understanding this correctly, the requestGuid param is the key we use to track the logging for this request. If so, could you please change the name of this variable to correlationId? This is the term most widely used and understood for this value. Thanks Siva.

Reply:
updated